http/file_server.ts

```diff
@@ -23,8 +23,6 @@ interface EntryInfo {
   name: string;
 }
 
-const encoder = new TextEncoder();
```

Comment:
Note: `Response` object take care of encoding, and this is unnecessary now. Nice clean up.